wrappers/java/src/main/java/com/evernym/sdk/vcx/proof/DisclosedProofApi.java

```diff
@@ -17,35 +17,6 @@ private DisclosedProofApi() {
     }
 
     private static final Logger logger = LoggerFactory.getLogger("DisclosedProofApi");
-    private static Callback vcxProofCreateCB = new Callback() {
-        @SuppressWarnings({"unused", "unchecked"})
-        public void callback(int commandHandle, int err, int proofHandle) {
-            logger.debug("callback() called with: commandHandle = [" + commandHandle + "], err = [" + err + "], proofHandle = [" + proofHandle + "]");
-            CompletableFuture<Integer> future = (CompletableFuture<Integer>) removeFuture(commandHandle);
-            if (!checkCallback(future, err)) return;
-            future.complete(proofHandle);
-        }
-    };
-
-    public static CompletableFuture<CreateProofMsgIdResult> proofCreate(
```

Comment:
Is there a problem with this API? Why is this deleted? I believe we are actually using this in our app.

Reply:
The one we use for creating a disclosed proof is https://github.com/AbsaOSS/libvcx/blob/f64f763c005465970e6bc05949b756db0bebaf48/wrappers/java/src/main/java/com/evernym/sdk/vcx/proof/DisclosedProofApi.java#L320. The method signature of `vcx_disclosed_proof_create_with_request` is https://github.com/AbsaOSS/libvcx/blob/f64f763c005465970e6bc05949b756db0bebaf48/libvcx/src/api/disclosed_proof.rs#L89. and https://github.com/AbsaOSS/libvcx/blob/f64f763c005465970e6bc05949b756db0bebaf48/wrappers/java/src/main/java/com/evernym/sdk/vcx/proof/DisclosedProofApi.java#L44 doesn't correspond with this. Only the Java wrapper has this obsolete method definition.